### PR TITLE
Removes wallshoves without an easy disarm replacement

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request
Removes wallshoving as a cheap and easy way to immediately flip the tide of battle. 

## Why It's Good For The Game
It makes those who have dangerous equipment more of a threat and discourages unarmed validhunters from challenging dangerous people. It mechanically encourages unarmed players to act believably and in accordance with the server's rules. 

## Testing Photographs and Procedure

I have not had time to meaningfully test yet. I covered the basic initial testing but I have run out of time for the moment and am more interested in getting immediate feedback on what will be a very controversial change. 

## Changelog
:cl:
balance: Shoving someone into a wall will no longer produce an immediate knockdown and item drop. 
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
image

Okay, per comments by both @mcmeiler and @PowerfulBacon, here's the third option I mentioned yesterday. 

If you just straight up don't like the concept at all, feel free to close this one as well, of course. I made this because of the cited reason for closing, not to try and force it.

**Comment by Mat05usz** [comment]
I'll copy over my comment from the closed PR:

> It makes those who have dangerous equipment more of a threat and discourages unarmed validhunters from challenging dangerous people. 

It gives an artificial advantage to people who have skill issues, causing them to think much less during combat.

> It mechanically encourages unarmed players to act believably and in accordance with the server's rules.

So now you want to be coding also according to rules? Rules are there to stop people from using certain aspects of code/game in situations that do not warrant it, not to 'force' coders to remove mechanics. Maybe make grenades unusable by non antags, because 'why the hell would they anyways', right?

Also I love this argument:

> act believably

What makes you, and a lot of other people, think that the only believable course of action for a human being is to run from any threat, basically being a coward? Just because some people IRL are, doesn't mean those in game have to be, especially considering that employing yourself on a goddamn space station takes a lot of courage, so if anything regular spessman should be more willing to fight, cause if he doesn't, he knows he's stranded on that small piece of metal orbitting in a distant system with a murderer that could've been easily stopped had he done anything.

**Comment by PowerfulBacon** [comment]
I think wall shoving should remain, just the dropping items removed.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	icon = 'icons/mob/human.dmi'
 	icon_state = ""
 	appearance_flags = KEEP_TOGETHER|TILE_BOUND|PIXEL_SCALE
+	var/faltering_grip = 0
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/human.dm:7]: Remove this

**Comment by Rukofamicom** [comment]
> I think wall shoving should remain, just the dropping items removed.

Tbh I'm not sure why I didn't think of it this way. The disarm is most of my issue with it, not the knockdown and slowdown

**Comment by mcmeiler** [comment]
> Okay, per comments by both @mcmeiler

I never said anything along these lines.

Simply removing it outright also takes depth out of the already shallow combat there is.
This turns any fight into mindless clicking until one person is horizontal with no thought put into it at all.

Personally what I think should be done is buff blocking levels on antag items significantly, to prevent people just shoving them over and winning immediately. Return armblade block while you're at it.

**Comment by DatBoiTim** [comment]
> I'll copy over my comment from the closed PR:
> 
> > It makes those who have dangerous equipment more of a threat and discourages unarmed validhunters from challenging dangerous people.
> 
> It gives an artificial advantage to people who have skill issues, causing them to think much less during combat.

Lemme get this straight.

People who get killed by others using a 1 click win have the skill issue, and not the people who have to rely on using 1 click wins?


> Simply removing it outright also takes depth out of the already shallow combat there is. 

I'm personally in favor of making shoves do stam damage, and wallshoves adding a multiplier to said stam damage.

Goonshoves are basically what I'd imagine. Removes the shovestuns, but keeps there a reward for positioning well.

> This turns any fight into mindless clicking until one person is horizontal with no thought put into it at all.

I'd argue we're already here. But this definitely won't help.

> Personally what I think should be done is buff blocking levels on antag items significantly, to prevent people just shoving them over and winning immediately. Return armblade block while you're at it.

I like the idea, but I think this is a better deterrent for under armed combatants as a whole. I would still support this in addition to the above idea I mentioned.

**Comment by mcmeiler** [comment]
> I'm personally in favor of making shoves do stam damage, and wallshoves adding a multiplier to said stam damage.

Ah yes, hardstunlocking via shoves. This is *somehow* the worst idea that was proposed yet.

**Comment by DatBoiTim** [comment]
> > I'm personally in favor of making shoves do stam damage, and wallshoves adding a multiplier to said stam damage.
> 
> Ah yes, hardstunlocking via shoves. This is _somehow_ the worst idea that was proposed yet.

I never said either had to do huge amounts of stamina damage. Could easily just make it do same damage as the base damage for harm intent(no improved shoves for lizards or whatever).

**Comment by Mat05usz** [comment]
> People who get killed by others using a 1 click win have the skill issue, and not the people who have to rely on using 1 click wins?

Yes, because the supposed 'one click win' is based only on aggressors unrobustness. Keep in mind that antag can also use the 'one click win', yet you forget bout that.

If you, as an antag, can't remember that you shouldn't be standing next to a wall while butchering your victim then I'm sorry, maybe you should get some skill.

**Comment by DatBoiTim** [comment]
> Yes, because the supposed 'one click win' is based only on aggressors unrobustness. Keep in mind that antag can also use the 'one click win', yet you forget bout that.

Yet you maintain that relying on 1 click wins makes you robust?

So to become skilled all one has to do is rely on advantages derived from poorly designed mechanics that amplify incredibly minute amounts of skill!

Excellent point you make. Time to nerf mining and xenobio again!

**Comment by Mat05usz** [comment]
> Yet you maintain that relying on 1 click wins makes you robust?

Nice strawman. Completely ignoring the second half of the sentence

**Comment by DatBoiTim** [comment]
> > Yet you maintain that relying on 1 click wins makes you robust?
> 
> Nice strawman. Completely ignoring the second half of the sentence

Using a fallacy doesn't mean my argument can be immediately discarded. [In fact doing so is a fallacy](https://rationalwiki.org/wiki/Fallacy_fallacy)

**Comment by Mat05usz** [comment]
> > > Yet you maintain that relying on 1 click wins makes you robust?
> > 
> > 
> > Nice strawman. Completely ignoring the second half of the sentence
> 
> Using a fallacy doesn't mean my argument can be immediately discarded. [In fact doing so is a fallacy](https://rationalwiki.org/wiki/Fallacy_fallacy)

Sure, cool link and all. I'll respond in the morning.

**Comment by DatBoiTim** [comment]
Anyways using skill issue as the crux of your argument when this is a newbie friendly server is incredibly counter-intuitive and counter-productive.

Relying on 1 click wins is also a skill issue. They aren't engaging to use, they aren't fun to be on the receiving end of, and they really aren't all that skillful, regardless of if there's a condition behind it or not.

The stations are incredibly cramped. Putting someone against a wall is ridiculously easy. This gets made easier considering the same intent can also forcefully move people.

I do think positional advantages should be rewarded. But an immediate stun/disarm should not be the reward.

**Comment by Rukofamicom** [comment]
> I do think positional advantages should be rewarded. But an immediate stun/disarm should not be the reward.

This is the supreme takeaway from all of the discussions my PRs have started and is what the PR I both expect and intend to see merged will be using as it's spirit.

> Anyways using skill issue as the crux of your argument when this is a newbie friendly server is incredibly counter-intuitive and counter-productive.

Also just want to parrot this and also say

Given Bee is supposed to be both RP-oriented and Newbie friendly, precision skill should not be nearly as huge of a factor in combat as it currently is. Skill should be a factor, but equipment and IC combat capability should have substantially more weight than they currently do by comparison.

It makes a good story when the lone, unarmed assistant single-handedly stops nukies once in a blue moon, but no matter how robust the assistant is that should be a very difficult feat to pull off even if the nukies are relatively new.

**Comment by Rukofamicom** [comment]
Also closing this for the time being, will make the final attempt from a fresh fork within a few days

**Pull request closed without merging**